bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/action/LGWebOSActions.java

```diff
@@ -88,8 +88,10 @@ private enum Button {
         LEFT,
         RIGHT,
         BACK,
+        EXIT,
         DELETE,
         ENTER,
+        ENTER_NAV,
```

Comment:
Perhaps this one should be named `ENTER` and the old one renamed to `ENTER_KEYBOARD` or something like that? The old one seems much less useful.

Actually I'm now wondering if instead we should preserve only the actions `DELETE` and `ENTER`, as all the other ones which maps to mouse actions are duplicated functionality since the same can be accomplished with channel `rcButton`. Of course the documentation should be updated to explain this. It seems like we now have a naming conflict when trying to fit two different things into one which doesn't need to be fitted as functionality is redundant?

Or maybe I'm missing something, but both solutions seems to end up in same place, `executeMouse()`?

Another approach could be separating this into two actions, `sendButton` and `sendKey` (or similar). The last one would accept only `DELETE` and `ENTER`, and first one could take all the others - without any naming conflicts. We could even consider skipping the validation, or adding an optional parameter for that, so that it would be possible to discover new buttons and use them.

Reply:
My intention was to break nothing.

> Another approach could be separating this into two actions, sendButton and sendKey (or similar). The last one would accept only DELETE and ENTER, and first one could take all the others - without any naming conflicts. We could even consider skipping the validation, or adding an optional parameter for that, so that it would be possible to discover new buttons and use them.

Now if we decide to break something, your proposal looks good to me.

Reply:
I now discover another **undocumented** action named `sendRCButton` which only uses the mouse API.
In fact, `sendButton` is an extension of `sendRCButton` including keyboard ENTER + DELETE and OK.

The action that is the equivalent of the rcButton channel is in fact `sendRCButton` except that the action is restricted to a limited choice of 8 predefined buttons while you can send any command value to the channel.

So `sendButton` could now do what `sendRCButton` is doing but without the restriction, and keeping the special case of OK. `sendRCButton` could be removed (was not documented). sendKey could be added to cover the keyboard API (ENTER+DELETE).

Reply:
Done